File: src/main/java/com/saybetter/domain/review/application/impl/RecordSymbolService.java
package com.saybetter.domain.review.application.impl;

import org.springframework.stereotype.Service;

import com.saybetter.domain.review.dao.repository.RecordSymbolReadRepository;
import com.saybetter.domain.review.dao.repository.RecordSymbolWriteRepository;

import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;

@Slf4j
@Service
@RequiredArgsConstructor
public class RecordSymbolService {

	private final RecordSymbolReadRepository recordSymbolReadRepository;
	private final RecordSymbolWriteRepository recordSymbolWriteRepository;

}

File: core/core-api/src/main/kotlin/io/say/better/domain/review/application/impl/RecordSymbolService.kt
package io.say.better.domain.review.application.impl

import io.say.better.storage.mysql.domains.review.entity.RecordSymbol
import io.say.better.storage.mysql.domains.review.repository.RecordSymbolReadRepository
import io.say.better.storage.mysql.domains.review.repository.RecordSymbolWriteRepository
import org.springframework.stereotype.Service

@Service
class RecordSymbolService(
    private val recordSymbolReadRepository: RecordSymbolReadRepository,
    private val recordSymbolWriteRepository: RecordSymbolWriteRepository,
) {
    fun createRecordSymbol(recordSymbol: RecordSymbol) {
        recordSymbolWriteRepository.save(recordSymbol)
    }
}
